### DISCUSSION TITLE
React Hook Form: Date Picker Not Focusing When Submitting with Errors

### DISCUSSION THREAD

**Discussion by TaMz07** [adobe/react-spectrum#5365] [Q&A]
```
<Controller
   control={control}
   name="expiresAt"
   defaultValue={parseDate(new Date().toISOString().split('T')[0])}
       render={({ field }) => {
              return (
                <DatePicker
                  label="Expires At"
                  errorMessage={errors.expiresAt?.message}
                  minValue={today(timeZone)}
                  {...field}
                />
              );
        }}
/>
```
    
   When submitting with an errors, the DatePicker not foucs

**Comment by snowystinger** [comment]
What version are you on? and are you using RAC/Hooks/RSP?

No matter which one you're using, could you try the latest nightly? We just had a bunch of work go into form handling, including auto focus of the first invalid field during an invalid form submission. https://github.com/adobe/react-spectrum/pull/5344

We're hoping to have this in our next release, but it'd be good to know if this solves your issue.

We also have some updated sections in our docs for integrating with React Hook Form. Those are a little harder to pre-release show you, but you could poke around in the upcoming docs here https://reactspectrum.blob.core.windows.net/reactspectrum/0b541f48848eb43f74ae57e8b21dc4455d95040b/verdaccio/docs/react-aria/forms.html#react-hook-form
